lib/improver/tests/metadata/test_forecast_times.py

```diff
@@ -0,0 +1,400 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# (C) British Crown Copyright 2017-2019 Met Office.
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# * Redistributions of source code must retain the above copyright notice, this
+#   list of conditions and the following disclaimer.
+#
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [standard import "from datetime import datetime, timedelta" should be placed before "import numpy as np"](https://app.codacy.com/app/metoppv_tech/improver/pullRequest?prid=4567561)